src/cov.jl

```diff
@@ -45,6 +60,15 @@ if VERSION < v"0.5.0-dev+679"
     Base.cov(x::DenseMatrix, wv::WeightVec; mean=nothing, vardim::Int=1) =
         scale!(scattermat(x, wv; mean=mean, vardim=vardim), inv(sum(wv)))
 
+
+"""
+    mean_and_cov(x, [wv,] [vardim=1]) -> (Vector, Matrix)
```

Comment:
The current docs use this `()` notation for returning tuples. See http://docs.julialang.org/en/latest/stdlib/collections/#Base.findmax!